stubs/redis/__init__.pyi

```diff
@@ -0,0 +1,6 @@
+# StrictRedis is defined as attribute in redis and mypy does not allow
+# imports of attributes. But to define stubs of redis we have to import
+# it. Thus to avoid mypy error, ignore is added here. 
+from redis import StrictRedis as RedisStrict  # type: ignore[attr-defined]
+
+StrictRedis = RedisStrict
```

Comment:
The current stubs of redis in typeshed are little ambiguous because it does not provide annotations for some methods and for some other methods it does not provide strict annotations ( just provide annotations with Any type).  To overcome this issue, we can use redis library itself because now redis have typed code.